base/essentials.jl

```diff
@@ -851,11 +851,11 @@ end
 
     Using `@inbounds` may return incorrect results/crashes/corruption
     for out-of-bounds indices. The user is responsible for checking it manually.
-    Only use `@inbounds` when it is certain from the information locally available
-    that all accesses are in bounds. In particular, using `1:length(A)` instead of
-    `eachindex(A)` in a function like the one above is _not_ safely inbounds because
-    the first index of `A` may not be `1` for all user defined types that subtype
-    `AbstractArray`.
+    Only use `@inbounds` when you are certain that all accesses are in bounds (as 
+    undefined behavior, e.g. crashes, might occur if this assertion is violated). In
+    particular, using `1:length(A)` instead of `eachindex(A)` in a function like
```

Comment:
```suggestion
    undefined behavior, e.g. crashes, might occur if this assertion is violated). For
    example, using `1:length(A)` instead of `eachindex(A)` in a function like
```

Reply:
That's unrelated to this PR, but sure. I agree it's an improvement.